repose-aggregator/core/repose-core/src/main/java/org/openrepose/nodeservice/httpcomponent/HttpComponentRequestProcessor.java

```diff
@@ -93,7 +93,7 @@ public static HttpUriRequest process(HttpServletRequest servletRequest,
     }
 
     private static URI getUri(HttpServletRequest servletRequest, URI target) throws URISyntaxException {
-        URIBuilder builder = new URIBuilder(target.toString() + servletRequest.getRequestURI());
+        URIBuilder builder = new URIBuilder(target);
```

Comment:
Is this a change in behavior or just a better way to do things? Does the `target` object contain the information in `servletRequest.getRequestURI()`?

Reply:
Yes, the `target` object contains the request path. This change was part of the routing refactor. See:
https://github.com/rackerlabs/repose/blob/REP-7231/repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeRoutingServlet.scala#L214
and:
https://github.com/rackerlabs/repose/blob/REP-7231/repose-aggregator/core/repose-core/src/main/scala/org/openrepose/powerfilter/ReposeRoutingServlet.scala#L194